module/input/NatNet/src/NatNet.cpp

```diff
@@ -64,22 +64,23 @@ namespace module::input {
                         .then("NatNet Data", [this](const UDP::Packet& packet) {
                             // Test if we are "connected" to this remote
                             // And if we are we can use the data
-                            if (remote == packet.remote.address && version != 0) {
+                            auto address = static_cast<uint32_t>(std::stoul(packet.remote.address));
```

Comment:
This isn't valid, the string is an IP address (something like `10.1.1.10` or maybe an IPv6 address) and using stoul will just extract the first number component and use it which is not correct here since it'll be just assuming that any IP address that starts with the same number is the same remote.
Instead remote should be a string and just store it in there.